### PR TITLE
WorldVistA Postinstaller

### PR DESCRIPTION
**Pull request by shabiel**
Inactivates old users; creates new users; updates intro text.

### CODE REVIEW

File: Common/wvDemopi.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Postinstall for OSEHRA VistA Plan 6
+# See IMPORTANT notes in the .m file.
```
Comment (ChristopherEdwards) [inline Common/wvDemopi.sh]: What important notes are these? This is the only mention of IMPORTANT in the change set.

File: README.md [hunk 2]
```diff
@@ -164,6 +164,51 @@ To capture the exported code from the container and remove the Docker objects, e
...
+    REPOSITORY           TAG                 IMAGE ID            CREATED             SIZE
+    wv                   latest              4d088aa275ff        13 hours ago        6.06GB
+    vehu                 latest              ed6175534a1c        20 hours ago        4.29GB
+    vxvista              latest              3866735fcfc0        21 hours ago        2.96GB
+    cachevista           latest              071386cb0e80        46 hours ago        15.4GB
+    centos               latest              75835a67d134        11 days ago         200MB
+    osehra/osehravista   latest              8d58b9b985d7        5 weeks ago         4.63GB
+    hello-world          latest              e38bc07ac18e        6 months ago        1.85kB
+
+So, if we want to push `wv` up, then we need to use the image ID `4d088aa275ff`
```
Comment (ChristopherEdwards) [inline README.md:185]: you can also shortcut this by doing docker build -t boo/wv:latest -t boo/wv:v3, etc.

**Review by ChristopherEdwards — APPROVED** [review]

**Comment by shabiel** [comment]
Ooops. Typo. I will remove it tomorrow.

**Comment by shabiel** [comment]
Done.

**Comment by shabiel** [comment]
That's a nice tip. I had no idea.

**Pull request merged**